config/migrations/20220511133621-add-publication-report-types.ttl

```diff
@@ -0,0 +1,54 @@
+@prefix mu: <http://mu.semte.ch/vocabularies/core/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix ext: <http://mu.semte.ch/vocabularies/ext/> .
+@prefix pub: <http://mu.semte.ch/vocabularies/ext/publicatie/> .
+
+@prefix PublicationReportTypeScheme: <http://themis.vlaanderen.be/id/concept-scheme/b75a85c3-a599-4d4f-a2a2-dea583f2fd86> .
+@prefix publicationReportType: <http://themis.vlaanderen.be/id/concept/publicatierapporttype/> .
+
+# PublicationReportTypes are only used to make the /publicaties/overzicht/rapporten route
+#   act more like any other route: fetching a (list of) models
+#   Every PublicationReportType stands for a certain combination of a type to group by and types to filter on.
+#   The definition of the filters and group options these types are done in the frontend.
+
+pub:Publicatierapporttype a ext:PublicClass .
+
+PublicationReportTypeScheme: a skos:ConceptScheme ;
+    mu:uuid "224637a8-a60a-4bcb-bd9b-f052456820a4" ;
+    skos:prefLabel "Publicatie-rapport-types" .
+
+publicationReportType:by-mandatees-on-decision-date
+    a skos:Concept ;
+    a pub:Publicatierapporttype ;
+    skos:inScheme PublicationReportTypeScheme: ;
+    mu:uuid "6f820cf1-cd36-49fb-ae82-b36ce5dbc6cb" .
+
+publicationReportType:by-mandatees-only-bvr
+    a skos:Concept ;
+    a pub:Publicatierapporttype ;
+    skos:inScheme PublicationReportTypeScheme: ;
+    mu:uuid "4e6cfd8c-c109-4117-ba06-93d07b03a357" .
+
+publicationReportType:by-mandatees-only-decree
+    a skos:Concept ;
+    a pub:Publicatierapporttype ;
+    skos:inScheme PublicationReportTypeScheme: ;
+    mu:uuid "9f059903-d47c-4584-86f3-bc9b16f9be5c" .
+
+publicationReportType:by-government-domains
+    a skos:Concept ;
+    a pub:Publicatierapporttype ;
+    skos:inScheme PublicationReportTypeScheme: ;
+    mu:uuid "b75c2a2f-2ebc-4999-94a3-48322e8fce6a" .
+
+publicationReportType:by-regulation-type
+    a skos:Concept ;
+    a pub:Publicatierapporttype ;
+    skos:inScheme PublicationReportTypeScheme: ;
+    mu:uuid "095a3da7-a383-48bf-b8c8-040ace0cad3c" .
+
+publicationReportType:by-regulation-type-only-not-via-council-of-ministers
+    a skos:Concept ;
+    a pub:Publicatierapporttype ;
```

Comment:
See comment above about casing. Would prefer this to be `PublicatieRapportType`. Thanks!